### PR TITLE
fix: add parameters to top level of screwdriver.yaml

### PR DESCRIPTION
**Pull request by adong**
## Context

This PR is successor of https://github.com/screwdriver-cd/data-schema/pull/349
which add `parameters` on top level of `screwdriver.yaml`
## Objective
Add `parameters` on top level of `screwdriver.yaml`

See `screwdriver.yaml` as example in here
https://github.com/adong/screwdriver-parameters-top-build/blob/604d2bbb7293a60cddef625a4c0e516d5e260099/screwdriver.yaml#L4

## References

POST
image

GET
image

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by tkyi** [comment]
eslint issue:
```
00:00:39 /sd/workspace/src/github.com/screwdriver-cd/data-schema/config/parameters.js
00:00:39   1:1  error  Expected newline after "use strict" directive  lines-around-directive
```

**Comment by adong** [comment]
Rebased with latest master

File: migrations/20191004-add_parameters_as_mediumtext-pipelines.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}pipelines`;
+
+module.exports = {
```
Comment (adong) [inline migrations/20191004-add_parameters_as_mediumtext-pipelines.js]: @parthasl Please take a look, as I'm not confident enough for DB migration

**Comment by adong** [comment]
Steps to generate a migration file: 

1. Follow https://github.com/screwdriver-cd/data-schema#usage, to install necessary dependencies 

2. Create a migration file with `datetime-performed_action-tableName.js`, and find example in either existing code base or the official [guide](https://sequelize.readthedocs.io/en/latest/docs/migrations/)

You can find [model definition](https://sequelize.readthedocs.io/en/latest/docs/models-definition/) and [data types](https://sequelize.readthedocs.io/en/latest/api/datatypes/#text) here.

Useful tutorial: https://medium.com/@andrewoons/how-to-define-sequelize-associations-using-migrations-de4333bf75a7

**Comment by tkyi** [comment]
Might be better to move the migration docs to another file and link to it in the README
otherwise the README looks really long and gets confusing (like when there's two `usage` sections back to back)

**Comment by adong** [comment]
Rebased with latest master

File: migrations/20191004-add_parameters_as_mediumtext-pipelines.js (hunk 1 above)
Comment (parthasl) [reply migrations/20191004-add_parameters_as_mediumtext-pipelines.js]: 👍

**Pull request merged**